packages/web-component/src/components/read-along-component/read-along.tsx

```diff
@@ -1014,40 +1057,59 @@ export class ReadAlongComponent {
    *
    * A sentence element with one or more words
    */
-  Sentence = (props: { words: Node[], attributes: NamedNodeMap }): Element => {
-    if (!this.hasTextTranslations && props.attributes["class"]) {
-      this.hasTextTranslations = props.attributes["class"].value.match("translation") != null;
+  Sentence = (props: { sentenceData: any | null }): Element => {
+    let words: ChildNode[] = Array.from(props.sentenceData.childNodes)
+    let attributes: NamedNodeMap = props.sentenceData.attributes
```

Comment:
Suggest using `hasAttribute()` and `getAttribute()` instead of this as they have more precise semantics for existing or non-existing attributes

Reply:
good point, thanks @dhdaines !